package/carbonio-mta

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+SERVICE_NAME="carbonio-mta"
+
+if [[ $(id -u) -ne 0 ]]; then
+  echo "Please run as root"
+  exit 1
+fi
+
+if [[ "$1" != "setup" ]]; then
+  echo "Syntax: ${SERVICE_NAME} <setup> to automatically setup the service"
+  exit 1;
+fi
+
+# decrypt the bootstrap token, asking the password to the sys admin
+# --setup check for SETUP_CONSUL_TOKEN env. variable and uses it
+# to avoid re-asking for the password multiple times
+echo -n "insert the cluster credential password: "
+export CONSUL_HTTP_TOKEN=$(service-discover bootstrap-token --setup)
+EXIT_CODE="$?"
+echo ""
+if [[ "${EXIT_CODE}" != "0" ]]; then
+  echo "cannot access to bootstrap token"
+  exit 1;
+fi
+# limit secret visibility as much as possible
+export -n SETUP_CONSUL_TOKEN
+
+POLICY_NAME="${SERVICE_NAME}-policy"
+POLICY_DESCRIPTION='Carbonio MTA service policy for config generation and for sidecar proxy'
```

Comment:
> 'Carbonio MTA service policy for config generation and for sidecar proxy'

We can omit the later 'for' in this sentence to make it more readable.
Consider: 'Carbonio MTA service policy for config generation and sidecar proxy'

Reply:
All in all, except for some text or descrption enhancements, everything looks good to me

Reply:
Dropped the last for ;) Marking it as resolved